pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -850,33 +855,41 @@ def _handle_distribution_tree(declarative_content):
                             package_id=declarative_content.content.pk,
                             modulemd_id=modulemd.content.pk,
                         )
-                        modulemd_pkgs.append(module_package)
+                        modulemd_pkgs_to_save.append(module_package)
 
                 for packagegroup in declarative_content.extra_data['group_relations']:
                     if not packagegroup.content._state.adding and content_already_saved:
                         group_pkg = PackageGroup_packages(
                             package_id=declarative_content.content.pk,
                             packagegroup_id=packagegroup.content.pk,
                         )
-                        group_pkgs.append(group_pkg)
+                        group_pkgs_to_save.append(group_pkg)
 
-        if modulemd_pkgs:
-            ModulemdPackages.objects.bulk_create(modulemd_pkgs, ignore_conflicts=True)
+        if modulemd_pkgs_to_save:
+            ModulemdPackages.objects.bulk_create(modulemd_pkgs_to_save, ignore_conflicts=True)
 
-        if group_pkgs:
-            PackageGroup_packages.objects.bulk_create(group_pkgs, ignore_conflicts=True)
+        if group_pkgs_to_save:
+            PackageGroup_packages.objects.bulk_create(group_pkgs_to_save, ignore_conflicts=True)
 
-        if category_groups:
-            PackageCategory_groups.objects.bulk_create(category_groups, ignore_conflicts=True)
+        if category_groups_to_save:
+            PackageCategory_groups.objects.bulk_create(
+                category_groups_to_save, ignore_conflicts=True
+            )
 
-        if env_groups:
-            PackageEnvironment_groups.objects.bulk_create(env_groups, ignore_conflicts=True)
+        if env_groups_to_save:
+            PackageEnvironment_groups.objects.bulk_create(env_groups_to_save, ignore_conflicts=True)
 
-        if env_optgroups:
+        if env_optgroups_to_save:
             PackageEnvironment_optionalgroups.objects.bulk_create(
-                env_optgroups, ignore_conflicts=True
+                env_optgroups_to_save, ignore_conflicts=True
             )
 
+        if update_collection_to_save:
+            UpdateCollection.objects.bulk_create(update_collection_to_save)
+
+        if update_record_collections_to_save:
```

Comment:
Naming is somewhat confusing and I don't have any better suggestions. Can you leave a comment that we are saving UpdateRecord -> UpdateCollection relations here? Or do you have any other ideas to clarify it?